docarray/array/storage/milvus/backend.py

```diff
@@ -271,11 +272,12 @@ def _docs_from_search_response(
     ) -> 'List[DocumentArray]':
         das = []
         for r in responses:
-            das.append(
-                DocumentArray(
-                    [Document.from_base64(hit.entity.get('serialized')) for hit in r]
-                )
-            )
+            da = []
+            for hit in r:
+                doc = Document.from_base64(hit.entity.get('serialized'))
+                doc.scores[f'score'] = NamedScore(value=hit.score)
```

Comment:
Is there a way we can know the score that was used? I think it is in the config, can we use that as the key here? how do other backends handle that?